paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetAnalyticsTest.kt

```diff
@@ -53,7 +53,6 @@ internal class PaymentSheetAnalyticsTest {
             response.testBodyFromFile("elements-sessions-requires_payment_method.json")
         }
 
-        validateAnalyticsRequest(eventName = "stripe_android.card_metadata_pk_available")
```

Comment:
I added these yesterday when I did the large refactor. I'm not exactly sure why or how it was there, or why it's gone now. But at least it's back to normal 🙈